backend_new/prisma/schema.prisma

```diff
@@ -0,0 +1,1025 @@
+}
+
+// Note: [rule] used to be a different enum but prisma doesn't support that kind of enum yet. See: https://github.com/prisma/prisma/issues/273
+model ApplicationFlaggedSet {
+  id                    String               @id() @default(dbgenerated("uuid_generate_v4()")) @db.Uuid
+  createdAt             DateTime             @default(now()) @map("created_at") @db.Timestamp(6)
+  updatedAt             DateTime             @updatedAt @map("updated_at") @db.Timestamp(6)
+  rule                  RuleEnum
+  ruleKey               String               @unique() @map("rule_key") @db.VarChar
+  resolvedTime          DateTime?            @map("resolved_time") @db.Timestamptz(6)
+  listingId             String               @map("listing_id") @db.Uuid
+  showConfirmationAlert Boolean              @default(false) @map("show_confirmation_alert")
+  status                FlaggedSetStatusEnum @default(pending)
+  resolvingUserId       String?              @map("resolving_user_id") @db.Uuid
+  userAccounts          UserAccounts?        @relation(fields: [resolvingUserId], references: [id], onDelete: NoAction, onUpdate: NoAction)
+  listings              listings             @relation(fields: [listingId], references: [id], onDelete: NoAction, onUpdate: NoAction)
+  applications          Applications[]
+
+  @@index([listingId])
+  @@map("application_flagged_set")
+}
+
+// Note: [phone_number] formerly max length 16; [label] formerly max length 256; [external_reference] formerly max length 4096
+model ApplicationMethods {
+  id                            String                      @id() @default(dbgenerated("uuid_generate_v4()")) @db.Uuid
+  createdAt                     DateTime                    @default(now()) @map("created_at") @db.Timestamp(6)
+  updatedAt                     DateTime                    @updatedAt @map("updated_at") @db.Timestamp(6)
+  type                          ApplicationReviewStatusEnum
+  label                         String?
+  externalReference             String?                     @map("external_reference")
+  acceptsPostmarkedApplications Boolean?                    @map("accepts_postmarked_applications")
+  phoneNumber                   String?                     @map("phone_number")
+  listingId                     String?                     @map("listing_id") @db.Uuid
+  listings                      listings?                   @relation(fields: [listingId], references: [id], onDelete: NoAction, onUpdate: NoAction)
+  paperApplications             PaperApplications[]
+
+  @@map("application_methods")
+}
+
+// Note: [additional_phone_number, additional_phone_number_type, household_size] formerly max length 16; 
+//       [contact_preferences, income] formerly max length 64;
+//       [app_url] formerly max length 256; 
+// Note: [contact_preferences] formerly max array length of 8
+// Note: [household_member] formerly had max array lenght of 32
+// Note: missing virtual [flagged] field
+model Applications {
```

Comment:
is this maybe supposed to be a singular `Application`?

Reply:
yeah I think this may be something we want to talk about, previously our naming convention was a bit all over the place (plural vs singular)

I wonder if you, @ludtkemorgan have a preference

I didn't think to establish a convention

Reply:
Hmm, I'm on the fence about this. Ideally it should be called `Application` since it is singular. However, I think the idea of this prisma change over is that the table names don't change (even if they are not the name we would have chosen today). It looks like we can change the name of the model but still keep the same table name with `@@map` at the bottom of the model. But then we run into the issue of the models don't match the table names for future reference. I could go either way on this. I'd prefer `Application`, but can see the argument for `Applications`. 

Let me know if I'm over thinking this